ide/atom/README.md

```diff
@@ -0,0 +1,47 @@
+[Packages](#packages) | [Shortcuts](#shortcuts) | [CLI](#cli) | [Feedback](#feedback)
+
+# Atom
+
+Download: <https://atom.io/>
+
+## Packages
+
+- busy-signal@1.3.0
+- editorconfig@2.2.2
+- fonts@2.0.0 (\*)
+- highlight-selected@0.13.1
+- intentions@1.1.2
+- linter@2.1.2
+- linter-eslint@8.1.4
+- linter-stylelint@3.10.0
+- linter-ui-default@1.2.2
+- minimap@4.27.1
+- pretty-json@1.6.3
+- react@0.16.2
```

Comment:
A couple of others that I have in my list that are kind of handy:
- atom-beautify (@0.29.17)
- pigments (@0.39.1)
- tabs-to-spaces (@1.0.3)